scipy/special/_support_alternative_backends.py

```diff
@@ -161,11 +227,12 @@ def __stdtr(df, t):
 
 
 def _stdtrit(xp, spx):
-    betainc = _get_native_func(xp, spx, 'betainc')  # noqa: F811
+    # Need either native stdtr or native betainc
+    stdtr = _get_native_func(spx, 'stdtr') or _stdtr(xp, spx)
     # If betainc is not defined, the root-finding would be done with `xp`
     # despite `stdtr` being evaluated with SciPy/NumPy `stdtr`. Save the
     # conversions: in this case, just evaluate `stdtrit` with SciPy/NumPy.
-    if betainc is None:
+    if stdtr is None:
```

Comment:
Note: this is purely for the sake of readability. There is no actual use case where a backend defines stdtr but not betainc.